### ISSUE TITLE
[Bug] Enabling a plugin does not carry over between sessions

### ISSUE THREAD

**Issue by Aaron-Bopp** [TfTHacker/obsidian42-brat#22]
When I enable a plugin it does not stick between sessions, if I close and open obsidian again it will be disabled. I don't think this is intended but if it is I would suggest changing the command to make it clear you are only enabling it for the current session. 

Here's my debug info
```
SYSTEM INFO:
	Obsidian version: v0.13.7
	Installer version: v0.12.19
	Login status: logged in
	Catalyst license: supporter
	Insider build toggle: on
	Base theme: dark
	Community theme: BRAT-Emrie-Candera----Bubble-Space-Theme
	Snippets enabled: 9
	Safe mode: off
	Plugins installed: 133
	Plugins enabled: 50
		1: Better footnote
		2: Advanced Tables
		3: Open vault in VSCode
		4: Dictionary
		5: Spaced Repetition
		6: Auto Link Title
		7: Templater
		8: Add Codemirror matchbrackets.js
		9: QuickAdd
		10: Hotkey Helper
		11: Readwise Official
		12: Pandoc Plugin
		13: Note Refactor
		14: Advanced new file
		15: Commits
		16: Wikipedia
		17: Tag Wrangler
		18: Collapse All
		19: Vault Statistics
		20: Cross-reference Navigation
		21: Gallery
		22: Fullscreen mode plugin
		23: Admonition
		24: Hider
		25: Sliding Panes (Andy's Mode)
		26: RSS Reader
		27: Adjacency Matrix Maker
		28: Emoji Shortcodes
		29: Obsidian Git
		30: Pane Relief
		31: txt as md
		32: Workspaces Plus
		33: Smart Random Note
		34: Activity History
		35: Linter
		36: Snippet Commands
		37: Dataview
		38: CustomJS
		39: Improved VimCursor
		40: Vimrc Support
		41: Obsidian42 - BRAT
		42: Preview Mode Keybinds
		43: Graph Analysis
		44: Better Word Count
		45: MySnippets
		46: Breadcrumbs
		47: MetaEdit
		48: Another Quick Switcher
		49: Various Complements
		50: Frontmatter Tag Suggest

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please try to make the issue happen in the help vault or disable community theme and snippets.
	Community plugins: for bugs, please try to make the issue happen in the help vault or disable community plugins.
```

**Comment by TfTHacker** [comment]
Thank you for your report. You are referring to the “enable plugin” command palette command? You enable it, but it does not stay enabled between sessions?

**Comment by Aaron-Bopp** [comment]
Yes, the `Enable plugin - toggle it on` command. Using it will enable whatever plugin, but if I close and reopen obsidian, it will no longer be enabled. The same behavior seems to exist for the disable plugin command.

**Comment by TfTHacker** [comment]
I am curious if you are still seeing this. No one else has reported the issue nor have I been able to duplicate it.

**Comment by Aaron-Bopp** [comment]
I am still experiencing this, even tried it on a new vault with no other plugins and the help vault. I think the reason comes from it not being added to `community-plugins.json` so it isn't loaded when I close and reopen obsidian. If I enable it manually, it shows up there, but if I enable it using the command, it doesn't. It's not that big of a deal, but it is weird.

**Comment by TfTHacker** [comment]
Let me take another stab at it :-)

**Comment by TfTHacker** [comment]
This is now resolved with 0.6.35 update to BRAT.
